firefox-ios/Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -899,7 +899,12 @@ class TabManagerMiddleware {
     private func addToBookmarks(_ shareItem: ShareItem?) {
         guard let shareItem else { return }
         // Add new mobile bookmark at the top of the list
-        profile.places.createBookmark(parentGUID: BookmarkRoots.MobileFolderGUID,
+        let isBookmarksRefactorEnabled = featureFlags.isFeatureEnabled(.bookmarksRefactor, checking: .buildOnly)
```

Comment:
We have this `BookmarksRefactorFeatureFlagProvider` defined in the project that you could use here instead. Nice for when you just want to blanket enable the thing in code.